app/controllers/shipit/api/tasks_controller.rb

```diff
@@ -12,6 +12,10 @@ def show
         render_resource stack.tasks.find(params[:id])
       end
 
+      def latest
+        render_resource stack.last_active_task
```

Comment:
Oh, I didn't remember we had a `/tasks` endpoint already. In this case do we even need a change?

You could hit `/index` with `?page_size=1`

Reply:
oh hm lemme see. would we be able to trust the ordering that this will always return the latest?

Reply:
Yes, it should be in descending order by default. If it isn't it's easily fixable.

Be careful though as it will return tasks, deploys and rollback altogether. So you might want to take more than 1 and filter out tasks.

Reply:
hm I don't really like that last part since I don't want to keep hitting the api and filtering the data until I get what I want. thinking about exposing a `latest_deploy` to just return the latest deploy/rollback task. thoughts?

Reply:
Then I'd suggest to add a `/deploys` endpoint that filter tasks out.

It's much more versatile that a `/tasks/latest` endpoint and much more homogenous with the rest of the API.

Reply:
works for me 👍

Reply:
There is a `Stack#deploys_and_rollbacks` relation. So it's basically just `render_resources stacl.deploys_and_rollbacks` plus the route.